### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cafferata**
Small typo fix #229

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ A PHP library for parsing, formatting, storing and validating international phon
      - [ShortNumberInfo](#shortnumberinfo)
      - [Mapping Phone Numbers to carrier](#mapping-phone-numbers-to-carrier)
      - [Mapping Phone Numbers to TimeZones](#mapping-phone-numbers-to-timezones)
- - [FQA](#fqa)
+ - [FAQ](#fqa)
```
Comment (giggsey) [inline README.md]: Can you fix the link too?

**Comment by giggsey** [comment]
Good spot, thanks.

Can you fix that additional typo from the review and I'll merge it.

**Comment by cafferata** [comment]
Of course! Bit of a shame that I have not seen it myself by sending the pull request.. 🙈

**Comment by giggsey** [comment]
Thanks for the contributions :)

**Pull request merged**